src/link-handler.js

```diff
@@ -82,10 +82,10 @@ export class DefaultLinkHandler extends LinkHandler {
     info.anchor = target;
     info.href = href;
 
-    let hasModifierKey = (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey);
+    let leftButtonClicked = event.which ? event.which === 1 : event.button & 1;
```

Comment:
This doesn't seem quite right to me. `event.which` and `event.button` are both supported by all browsers we need to support, so only one should be necessary. The boolean `&` on `event.button` probably needs to be a strict comparison against `0`, which represents the main mouse button.

I think we do need to keep the checks for the various meta keys too, so that ctrl + click is handled properly.